zipkin-server/src/main/java/zipkin/server/ZipkinServer.java

```diff
@@ -30,8 +29,7 @@
 
   public static void main(String[] args) {
     new SpringApplicationBuilder(ZipkinServer.class)
-        .listeners(new RegisterZipkinHealthIndicators())
```

Comment:
I had to move this to resource-based loading (like we do for add-ons.. spring.factories) else it wouldn't trigger during tests.